src/components/Form/Controls/Toggle.tsx

```diff
@@ -36,7 +36,7 @@ export class Toggle extends Base<Props> {
   @Model('whether toggle is checked', { event: 'input', default: false, type: Boolean })
   public on!: boolean;
 
-  private currentOn: boolean = false;
+  private currentOn: boolean = this.on;
```

Comment:
I set it to false as there is watch for on, and that time going to set the currentOn = this.on